src/task/SubscribeAndReceiveSubscribeAckTask.ts

```diff
@@ -107,6 +108,12 @@ export default class SubscribeAndReceiveSubscribeAckTask extends BaseTask {
         }
 
         handleSignalingClientEvent(event: SignalingClientEvent): void {
+          if (event.isConnectionTerminated()) {
+            this.logger.error('SubscribeAndReceiveSubscribeAckTask connection terminated');
+            this.cancel();
```

Comment:
Will this trigger a reconnect?  Can we comment what we are doing here.

Reply:
Sure, I think I need to modify a little bit here to make it more clear. I will add comment.